src/main/java/teammates/storage/api/FeedbackResponseCommentsDb.java

```diff
@@ -122,6 +122,17 @@ public FeedbackResponseCommentAttributes getFeedbackResponseComment(
         return makeAttributes(getFeedbackResponseCommentEntitiesForResponse(feedbackResponseId));
     }
 
+    /**
+     *  Gets all response comments for a response from participant.
+     */
+    public List<FeedbackResponseCommentAttributes> getFeedbackResponseCommentForResponseFromParticipant(
+            String feedbackResponseId, boolean isFromParticipant) {
+        Assumption.assertNotNull(Const.StatusCodes.DBLEVEL_NULL_INPUT, feedbackResponseId);
+        Assumption.assertNotNull(Const.StatusCodes.DBLEVEL_NULL_INPUT, isFromParticipant);
```

Comment:
I think `isFromParticipant` can't be null since it's a primitive `boolean` which makes this line unnecessary?